mlflow/store/tracking/abstract_store.py

```diff
@@ -234,29 +234,37 @@ def restore_run(self, run_id):
         """
         pass
 
-    def create_trace(self, experiment_id, start_time, end_time, status, attributes, tags):
+    def create_trace(
+        self,
+        experiment_id,
+        timestamp_ms,
+        execution_time_ms,
+        status,
+        request_metadata,
+        tags,
+    ):
```

Comment:
Nit: Can we add type hints while we're here?

Reply:
Since this whole file does not use type hint, not adding type hint looks better to me. I added type in the docstring section. 

Btw personally I am a hater of type hint, and prefer putting type in the comment because of the development overhead. I remember we did a survey among Tensorflow developers, and it's about 5:5 lovers vs haters.

Reply:
Can we add type hints? Much of this module was written before type hints existed. Let's start now :). It helps developers implement the spec.